src/Parse/Bracket.php

```diff
@@ -35,7 +35,7 @@ public function setOptions($options = [])
      * @param  string $template
      * @param  array $vars
      * @param  array $options
-     * @return self
+     * @return bool|string|string[]|null
```

Comment:
This would only return bool or string from what I can tell.

Reply:
The `string` or `array`  is referenced from [str_replace](https://www.php.net/manual/en/function.str-replace.php)  and `null` from [preg_replace ](https://www.php.net/manual/en/function.preg-replace.php) - See `parseKeyBooleans` - we can cast the returned value to string. Should I make the changes?

Reply:
Array is not going to be returned because str_replace is not being provided an array as input. Null is unlikely to be returned as it only gets returned by preg_replace if an error occurs, which is unlikely. If you want to include it, then what should actually happen is that the code should properly handle that return value so it doesn't get passed up the call stack and we should add a unit test specifically for whatever input would have to be provided to return that value.

Reply:
Seems unnecessary. Just replacing self with boolean and string.